packages/arb-validator-core/gobridge/messageTester.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
None of this class needs to exist. The EthBridge messageTester is only used for internal testing of the EthBridge and isn't needed in the GoBridge